stored_requests/fetcher_test.go

```diff
@@ -243,7 +243,7 @@ func TestAccountCacheHit(t *testing.T) {
 		})
 
 	metricsEngine.On("RecordAccountCacheResult", metrics.CacheHit, 1)
-	account, errs := aFetcherWithCache.FetchAccount(ctx, "known")
+	account, errs := aFetcherWithCache.FetchAccount(ctx, json.RawMessage("{}"), "known")
```

Comment:
Might be a good idea to add another test case with some default account JSON

Reply:
This test cases is for account cache. The flow does not reach the account json merge code, as account fetcher is mocked.